dist/style-guide/item-general-typography.html

```diff
@@ -120,19 +120,16 @@ <h2 class="kss-title kss-title--level-2">
                             <div class="kss-description">
                 <ul>
 <li>Avoid small font sizes.</li>
-<li>
-  Use relative units for font size, such as ems or rems. While modern browsers can smoothly zoom
-  pixel-based layouts, sizing type in relative units ensures an entire layout can be scaled up or
-  down by simply updating the font-size of the body element.
-</li>
-<li>A design should allow typography to be magnified up to 200% by the user without clipping or distorting content.
-</li>
- <li>Select basic, simple, easily-readable fonts and use a limited number of fonts.</li>
- <li>Avoid small font sizes and use relative units for font size (ems or rems).</li>
- <li>Limit the use of font variations such as bold, italics, and ALL CAPITAL LETTERS (caps are similar to screaming to screenreaders).</li>
- <li>Don't rely only on the appearance of the font (color, shape, font variation, placement, etc.) to convey meaning.</li>
- <li>Avoid blinking or moving text. </li>
- <li>Use real text rather than text within graphics.</li>
+<li>Use relative units for font size, such as ems or rems. While modern browsers can smoothly zoom
+pixel-based layouts, sizing type in relative units ensures an entire layout can be scaled up or
+down by simply updating the font-size of the body element.</li>
+<li>A design should allow typography to be magnified up to 200% by the user without clipping or distorting content.</li>
+<li>Select basic, simple, easily-readable fonts and use a limited number of fonts.</li>
+<li>Avoid small font sizes and use relative units for font size (ems or rems).</li>
```

Comment:
Could probably cut out "and use relative..." since the first `li` already notes this.